yaml/Dockerfile

```diff
@@ -1,11 +1,15 @@
 FROM    alpine:3.5
```

Comment:
I had that at some point. Will reevaluate but right now having problems with some dependencies requiring `gcc`